biogeochem/EDPhysiologyMod.F90

```diff
@@ -2026,7 +2027,7 @@ subroutine recruitment( currentSite, currentPatch, bc_in )
     !----------------------------------------------------------------------
 
     allocate(temp_cohort) ! create temporary cohort
-    call zero_cohort(temp_cohort)
+    call temp_cohort%ZeroValues()
```

Comment:
@rgknox

Reply:
just noting that @adrifoster and I checked in on this yesterday, and noted that since the two if conditions are mutually exclusive, it can be done either way.